internal/api/v1beta1/v1beta1.go

```diff
@@ -23,8 +24,11 @@ type Handler struct {
 	ruleService      RuleService
 }
 
-func Register(ctx context.Context, s *server.MuxServer, gw *server.GRPCGateway, deps api.Deps) {
-	gw.RegisterHandler(ctx, shieldv1beta1.RegisterShieldServiceHandlerFromEndpoint)
+func Register(ctx context.Context, address string, s *grpc.Server, gw *runtime.ServeMux, deps api.Deps) error {
+	err := shieldv1beta1.RegisterShieldServiceHandlerFromEndpoint(ctx, gw, address, []grpc.DialOption{grpc.WithInsecure()})
```

Comment:
Can we avoid using deprecated API (`grpc.WithInsecure()`)?

Reply:
Changed it
`[]grpc.DialOption{grpc.WithTransportCredentials(insecure.NewCredentials())}`